docs/BondingCurve.md

```diff
@@ -0,0 +1,170 @@
+- Bonded Token
+- Dividend Pool
+- Buy Curve Logic
+- Sell Curve Logic
+
+## Usage
+
+### Bonding Curve
+
+The primary point of external interaction with the curve - this is where users can buy & sell bondedTokens, and where the curve recieves revenue.
+
+[**`priceToBuy`**](./contracts/BondingCurve/BondingCurve.sol): Determine the current price in collateralTokens to buy a given number of bondedTokens.
+
+```
+function priceToBuy(
+ uint256 numTokens
+) public
+```
+
+[**`rewardForSell`**](./contracts/BondingCurve/BondingCurve.sol): Determine the current payout in collateralTokens to sell a given number of bondedTokens.
+
+```
+function rewardForSell(
+ uint256 numTokens,
+) public
+```
+
+[**`buy`**](./contracts/BondingCurve/BondingCurve.sol): Buy a given number of bondedTokens with an amount of collateralTokens determined by the current rate from the buy curve. The caller can specify the maximum total price in collateralTokens they're willing to pay, and a recipient address to transfer the new bondedTokens to.
+
+The appropriate amount of collateralTokens must have previously been approved for the bonding curve contract by the caller.
+
+```
+function buy(
+ uint256 numTokens,
+ uint256 maxPrice,
+ address recipient
+) public
+```
+
+[**`sell`**](./contracts/BondingCurve/BondingCurve.sol): Sell a given number of bondedTokens for an amount of collateralTokens determined by the current rate from the sell curve. The caller can set a minimum total value of collateralTokens they're willing to sell for, and a recipient to transfer the proceeds of the sale to.
+
+```
+function sell(
+ uint256 numTokens,
+ uint256 minPrice,
+ address recipient
+) public
+```
+
+[**`pay`**](./contracts/BondingCurve/BondingCurve.sol): Pay the beneficiary in collateralTokens. Revenue send in this method is distributed between the beneficiary and the bondedToken holders according to the dividendPercentage parameter;
+
+```
+function pay(
+ uint256 amount
+) public
+```
+
+### Dividend Pool
+
+Users interact with this contract to claim their dividend allocations.
+
+[**`withdraw`**](./contracts/BondingCurve/BondingCurve.sol): Withdraw collateralToken dividends sender is entitled to for a given period, in blocks.
+
+```
+function withdraw(
+ uint start,
+ uint end
+) public
+```
+
+## Future Plans
+
+We envision the following features may be useful to organizations implementing bonding curves.
+
+### Financial Features
+
+- **Hatching** - An initial buying phase where selling is disabled up until a certain amount of tokens are bought. This helps ensure a certain amount of return for early investors.
+- **Vesting** - Vesting periods can be added to minted tokens, which helps fight against pumping and dumping.
+- **Taxes** - A % fee for selling back to the market can be added to encourage secondary market trading.
+- **Governance via BondedTokens** - Voting power can be granted to token holders, which can help further insulate their potentially risky investment.
```

Comment:
Suggested change: `which can help further insulate their potentially risky investment.` to `which gives investors a say in how their money is being used.`